client/consensus/src/lib.rs

```diff
@@ -103,6 +106,35 @@ where
     authorities
 }
 
+/// Return the set of authorities assigned to the paraId where
```

Comment:
please update the variable names (you are returning something called authorities haha). Also the comment

Reply:
True, I was also thinking about renaming `get_authorities_from_orchestrator` into something more generic, because now we fetch authorities and slot freq

Reply:
yes we should, maybe something like get consensus data?